pandas/tests/test_strings.py

```diff
@@ -1008,6 +1008,13 @@ def test_replace(self):
                     values = klass(data)
                     pytest.raises(TypeError, values.str.replace, 'a', repl)
 
+    def test_replace_single_pattern(self):
+        values = Series(['abc','123'])
```

Comment:
Can you add a comment for the issue (`# GH 24804`)

Reply:
Yes